main.go

```diff
@@ -35,7 +36,23 @@ func main() {
 	}
 	utils.KeployVersion = version
 	fmt.Println(logo, " ")
-	fmt.Printf("version: %v\n\n", version)
+	releaseInfo, err2 := utils.GetLatestGitHubRelease()
+	if err2 != nil {
+		log.Debug("Failed to fetch latest release version", err2)
+	}
+	updatetext := models.GrayString.Sprint("keploy update")
```

Comment:
Why are you still showing this box when err has occurred? We have to skip this right..?

Reply:
Refactored the code to reflect the same